back/spec/services/survey_results_generator_service_spec.rb

```diff
@@ -157,14 +157,21 @@
           question: { 'en' => 'What is your favourite colour?' },
           required: false,
           totalResponses: 4,
-          customFieldId: text_field.id
+          customFieldId: text_field.id,
+          textResponses: a_collection_containing_exactly(
+            { answer: 'Red' },
+            { answer: 'Blue' },
+            { answer: 'Green' },
+            { answer: 'Pink' }
+          )
```

Comment:
This was a tricky one to fix in the end - turns out rspec can't actually ignore order in nested hashes as I thought it would, so I had to change all the matchers to be `match` instead of `eq` and wrap the array with this method

Reply:
Thanks, James! I would have probably spent a lot more time on this without you 🙌